docs/reference/commandline/container_restart.md

```diff
@@ -22,3 +22,33 @@ Restart one or more containers
 ```console
 $ docker restart my_container
 ```
+
+
+### <a name="signal"></a> Stop container with signal (-s, --signal)
+
+The `--signal` flag sends the system call signal to the container to exit.
+This signal can be a signal name in the format `SIG<NAME>`, for instance
+`SIGKILL`, or an unsigned number that matches a position in the kernel's
+syscall table, for instance `9`.
+
+The default signal to use is defined by the image's [`StopSignal`](https://github.com/opencontainers/image-spec/blob/v1.1.0/config.md),
+which can be set through the [`STOPSIGNAL`](https://docs.docker.com/reference/dockerfile/#stopsignal)
+Dockerfile instruction when building the image, or configured using the the
```

Comment:
```suggestion
Dockerfile instruction when building the image, or configured using the
```

Reply:
Ah dang! I wish some of the linters would also run on markdown; `dupword` would've caught this one; https://github.com/docker/cli/blob/c767b9b2dacaa344e30426c2fcd65f7440392aa7/.golangci.yml#L6